examples/mnist/notebook.jl

```diff
@@ -87,18 +87,19 @@ end
 # is controlled using using the `finaliser` hyperparameter of the
 # classifier.
 
-# We now define the MLJ model. If you have a GPU, substitute
-# `acceleration=CUDALibs()` below:
+# We now define the MLJ model.
 
 ImageClassifier = @load ImageClassifier
-clf = ImageClassifier(builder=MyConvBuilder(3, 16, 32, 32),
-                      batch_size=50,
-                      epochs=10,
-                      rng=123)
+clf = ImageClassifier(
+    builder=MyConvBuilder(3, 16, 32, 32),
+    batch_size=50,
+    epochs=10,
+    rng=123,
+)
 
 # You can add Flux options `optimiser=...` and `loss=...` here. At
```

Comment:
```suggestion
# You can add to the constructor Flux options `optimiser=...` and `loss=...` here. At
```
because its later mention `To run on a GPU, set acceleration=CUDALib() and omit rng.` which if followed by a call to `machine` (so one may think they should add it the machine and not the constructor if that's not explicit.)

Reply:
Addressed in a new commit.